src/main/groovy/com/google/appengine/AppEnginePlugin.groovy

```diff
@@ -245,28 +256,32 @@ class AppEnginePlugin implements Plugin<Project> {
         }
     }
 
-    private void configureAppConfig(Project project, AppEnginePluginExtension appEnginePluginExtension) {
+    private void configureWorkingDirectory(Project project) {
+        AppConfigTaskTemplate.metaClass.getWorkingDirectory = { -> this.explodedAppDirectory }
```

Comment:
So I didn't think there would be automatic getter/setter creation for private members in groovy, but if there's some naming ambiguity maybe we should change things around instead of letting the ambiguity persist.  Perhaps changing the static methods from "get" to something that doesn't clash here.